Tests/MongoCoreTests/ProtocolTests.swift

```diff
@@ -67,9 +67,9 @@ class ProtocolTests: XCTestCase {
         _ = try OpMessage(reading: &buffer, header: header)
     }
     
-    func testOpMessageDeniesFirstUInt16Flags() throws {
-        XCTAssertNoThrow(try OpMessage(reading: &buffer, header: header))
-    }
+    // func testOpMessageDeniesFirstUInt16Flags() throws {
```

Comment:
This is a pretty bad piece of code. But it's really an non-critical test. Let's add a TODO comment at least